PowerEditor/src/MISC/RegExt/regExtDlg.cpp

```diff
@@ -48,8 +48,8 @@ const TCHAR defExtArray[nbSupportedLang][nbExtMax][extNameMax] =
 	{TEXT("web script"),
 		TEXT(".html"), TEXT(".htm"), TEXT(".shtml"), TEXT(".shtm"), TEXT(".hta"),
 		TEXT(".asp"), TEXT(".aspx"),
-		TEXT(".css"), TEXT(".js"), TEXT(".json"), TEXT(".jsm"), TEXT(".jsp"),
-		TEXT(".php"), TEXT(".php3"), TEXT(".php4"), TEXT(".php5"), TEXT(".phps"), TEXT(".phpt"), TEXT(".phtml"),

```

Comment:
.phtml "was" the standard file extension for PHP 2 programs, doesn't mean it doesn't exist anymore.
I could imagine there are still some servers use php2 so some phtml files still exist. 
OTOH, there's no reason to remove it if we keep php3 & php4 which are taken over by php5.

Reply:
Come on, PHP 2 has been released on 1997! It's so old it's not even listed on the [Unsupported Historical Releases
](https://www.php.net/releases/index.php) page lol.

Here I'm just applying the [Less is more](https://en.wikipedia.org/wiki/Less_is_more) principle, removing a tiny bit of low-hanging bloat, to improves usability of the relevant parts.

Reply:
My point is phtml could still be used.